### PR TITLE
PR for task : Add your photo to the GCI FOSSASIA Site Student Section

### PR DESCRIPTION
**Pull request by adityak1121**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Added and Updated my credentials to students.yml
- Added and Updated my image adityak1121-WebDev2.png

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # New members please add your credentials to a random position in the file.
 # This is to remove conflicts with other people adding their credentials.
 
+- name: Aditya Kumar
+  github: adityak1121
+  image: adityak1121-WebDev.png
+  twitter: adityak11212
+  facebook: Aditya Kumar
```
Comment (abishekvashok) [inline _data/students.yml]: You got your facebook username wrong please go to your profile and put the thing after fb.com/ here ita your username pls.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 1 above)
Comment (adityak1121) [reply _data/students.yml]: like this https://www.facebook.com/adityak1121 ?

Comment (abishekvashok) [reply _data/students.yml]: Nope only the part after facebook.com/ that is adityak1121

**Review by abishekvashok — APPROVED** [review]

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Dismissed

**Comment by adityak1121** [comment]
Do I have to change these?
 "Read and understood (see CONTRIBUTING.md)
 Included a live link or preview screenshot
 Images are 240 x 240 [w x h]
 Resolved merge conflicts
 Included a description of my change below"

**Pull request merged**